include/z64math.h

```diff
@@ -22,7 +22,7 @@ typedef struct {
     /* 0x0 */ f32 x;
     /* 0x4 */ f32 y;
     /* 0x8 */ f32 z;
-} Vec3f; // size = 0xC
+} xyz_t; // size = 0xC
```

Comment:
This and the `Vec3s` -> `s_xyz` change will be a bit though of a change

Reply:
Tough? Thought?

Reply:
tough, yes :sweat_smile: